examples/main.tf

```diff
@@ -0,0 +1,47 @@
+data "aws_iam_policy_document" "read-a-bucket" {
+  statement {
+    sid    = "AllowReadOfABucket"
+    effect = "Allow"
+
+    actions = [
+      "s3:GetObject",
+    ]
+
+    resources = [
+      "arn:aws:s3:::public-bucket/*",
+    ]
+  }
+}
+
+module "periodic_worker" {
```

Comment:
I think the example should also use variables.tf instead of hardcoding in the main.tf

Reply:
what do you mean by using variables.tf? In the example, it sets the value of the variables.